dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/specific/frameworkregistryimports/FrameworkRegistryImportsUpdater.kt

```diff
@@ -23,9 +24,18 @@ class FrameworkRegistryImportsUpdater {
         }!!
 
         val freeMarkerContext = mapOf(
-            "frameworks" to allRegisteredFrameworks.map { it.name },
+            "frameworks" to allRegisteredFrameworks.map {
+                mapOf(
+                    "identifier" to it.name,
+                    "baseNameInCamelCase" to getNameFromLabel(it.name),
+                )
+            },
         )
+        writeIntoRegistryTsFiles(repository, freeMarkerContext)
```

Comment:
why is this new? was it forgotten before?

Reply:
or is it just refactored?

Reply:
Ich musste einen Teil der `update()` Funktion in eine eigene Funktion auslagern, weil Sonar gemeckert hat dass `update()` mehr als 30 Zeilen hat.

Und `update()` hatte jetzt nur plötzlich mehr als 30 Zeilen, weil durch meine Umbenennungen manche Dinge nicht mehr in eine Zeile gepasst haben, und die Funktion dadurch aufgebläht wurde.

Also reines refactoring an dieser Stelle.